operations/validate_csm_health.md

```diff
@@ -61,335 +54,100 @@ Health Check scripts can be run:
-
-   * If after several tries one or more BGP session remains **Idle**, see [Check BGP Status and Reset Sessions](network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md)
-
-
-1. Repeat the above **Aruba** procedure using the second peer-address (10.252.0.5 in this example).
-
-<a name="net-kea"></a>
-### 1.4 Verify that KEA has active DHCP leases
-
-Verify that KEA has active DHCP leases. Right after an fresh install of CSM, it is important to verify that KEA is currently handing out DHCP leases on the system. The following commands can be run on any of the master nodes or worker nodes.
-
-Get an API Token:
-```bash
-ncn# export TOKEN=$(curl -s -S -d grant_type=client_credentials \
-                 -d client_id=admin-client \
-                 -d client_secret=`kubectl get secrets admin-client-auth \
-                 -o jsonpath='{.data.client-secret}' | base64 -d` \
-                          https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
-```
-
-Retrieve all the leases currently in KEA:
-```bash
-ncn# curl -H "Authorization: Bearer ${TOKEN}" -X POST -H "Content-Type: application/json" -d '{ "command": "lease4-get-all", "service": [ "dhcp4" ] }' https://api-gw-service-nmn.local/apis/dhcp-kea | jq
-```
-
-If there is an non-zero amount of DHCP leases for air-cooled hardware returned, that is a good indication that KEA is working.
-
-<a name="net-extdns"></a>
-### 1.5 Verify ability to resolve external DNS
-
-If unbound is configured to resolve outside hostnames, then the following check should be performed. If unbound is not configured to resolve outside hostnames, then this check may be skipped.
-
-Run the following on one of the master or worker nodes (not the PIT node):
-
-```bash
-ncn# nslookup cray.com ; echo "Exit code is $?"
-```
-
-Expected output looks similar to the following:
-```
-Server:         10.92.100.225
-Address:        10.92.100.225#53
-
-Non-authoritative answer:
-Name:   cray.com
-Address: 52.36.131.229
-
-Exit code is 0
-```
-
-Verify that the command has exit code 0, reports no errors, and resolves the address.
-
-<a name="net-spire"></a>
-### 1.6 Verify Spire Agent is Running on Kubernetes NCNs
-
-Execute the following command on all Kubernetes NCNs (all worker nodes and master nodes), excluding the PIT node (booted from the LiveCD):
+And the Kubernetes test suite via:
 
 ```bash
-ncn# goss -g /opt/cray/tests/install/ncn/tests/goss-spire-agent-service-running.yaml validate
-```
+ncn# /opt/cray/tests/install/ncn/automated/ncn-kubernetes-checks
+```
+Review the output for `Result: FAIL` and follow the instructions provided to resolve any such test failures. With the exception of the [Known Test Issues](#autogoss-issues), all health checks are expected to pass.
+
+The ncn-healthcheck suite covers the following (as appropriate for worker, master and storage nodes):
+* Commands Available
+* Interfaces Has IP
```

Comment:
"Interfaces Have IP addresses" ?

Reply:
Per the comment below, I will remove the list of tests